<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVRModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerVRModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="initializePushButton">
     <property name="text">
      <string>Initialize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startVRPushButton">
     <property name="text">
      <string>StartVR</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stopVRPushButton">
     <property name="text">
      <string>StopVR</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
